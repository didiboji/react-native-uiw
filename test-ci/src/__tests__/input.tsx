import 'react-native';
import React from 'react';
import Input from '../lib/Input';
import { render, fireEvent } from '@testing-library/react-native';

describe('input', () => {
  it('snapshot', () => {
    const DefaultInput = () => {
      return <Input testID="input" />;
    };
    // const { toJSON } = render(<DefaultInput />);
    // expect(toJSON()).toMatchSnapshot();
  });
});
