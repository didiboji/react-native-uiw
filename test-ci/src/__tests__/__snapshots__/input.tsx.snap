// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input snapshot 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flexDirection": "row",
        "height": 30,
        "paddingVertical": 0,
      },
      undefined,
    ]
  }
>
  <View
    style={
      [
        {
          "alignItems": "center",
          "backgroundColor": "transparent",
          "flexDirection": "row",
          "height": "100%",
          "marginBottom": 0,
          "marginTop": 0,
        },
        {
          "borderColor": "#ccc",
          "flex": 1,
        },
        {
          "borderBottomWidth": 1,
        },
      ]
    }
  >
    <TextInput
      editable={true}
      onChangeText={[Function]}
      onFocus={[Function]}
      style={
        [
          {
            "fontSize": 14,
          },
          {
            "backgroundColor": "transparent",
            "color": "#000",
            "flex": 1,
            "paddingVertical": 0,
          },
          [],
        ]
      }
      testID="input"
    />
  </View>
</View>
`;
